shared/data.js

```diff
@@ -359,7 +359,7 @@ const units = {
         'attackSpeed': 0,
         'damage': 0,
         'description': 'Requires Automation Factory. The Reaver explodes when dying, dealing 50 damage to all units range 1, and 25 range 2.',
-        'health': 20,
+        'health': 30,
```

Comment:
Lowering Reaver cost might make it the most viable to stack at least 1 health now, since 1 Reaver could take out potentially 5 marines for about the same cost LOL but quickly becomes useless if everyone builds 1 health.

Reply:
i was thinking 500 is just way too much, because odds are you aren't going to be able to kill 2 enemy marines every time your reaver dies, so making it cost efficient is nearly impossible

Reply:
Yeah agreed with 500 being too much, could be good to stop early pushes then become irrelevant later on as people stack health, similar place as turrets so I think this could work